src/puppet-oa.ts

```diff
@@ -535,7 +532,8 @@ class PuppetOA extends Puppet {
     conversationId: string,
     file     : FileBox,
   ): Promise<string> {
-    return this.messageSend(conversationId, file)
+    const msgtype: OAMediaType = (file.mimeType === 'image/jpeg') ? 'image' : 'voice'
```

Comment:
Can we make this logic more robust?

I'd like to suggest that:

1. we do a `switch` for checking `file.mimeType` and set the `MsgType` correctly
2. we throw an exception for the unknown `mimeType` instead of failing silently

Reply:
thanks for the suggestions, have updated